devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java

```diff
@@ -78,13 +78,18 @@ public FileCollection getClasspath() {
     }
 
     @OutputFile
-    public File getOutputDir() {
+    public File getRunnerJar() {
         return new File(getProject().getBuildDir(), extension().finalName() + "-runner.jar");
     }
 
+    @OutputFile
+    public File getFastJar() {
+        return new File(getProject().getBuildDir(), "quarkus-app");
```

Comment:
Sorry my ignorance, but isn't a `·jar' extension missing here?

Reply:
This is just the directory where all the fast-jar output goes

Reply:
This should be `@OutputDirectory` then. 
BTW, reading the documentation, a missing directory should not be a problem.